lib/Core/ConsoleAnalytics.js

```diff
@@ -0,0 +1,50 @@
+import defined from "terriajs-cesium/Source/Core/defined";
```

Comment:
Is this file restored because the git auto-merge didn't realize it was renamed when being TSified?

Reply:
Good catch. Fixed.